docs/docs/csv-connector-columns.md

```diff
@@ -11,7 +11,7 @@
 | Access Level        | "dataset value" or "access"                                                                     |
 | Start Date          | "start date"                                                                                    |
 | End Date            | "end date"                                                                                      |
-| License             | "license type" or "licensing"                                                                   |
+| Licence             | "licence type" or "licensing"                                                                   |
```

Comment:
Is the CSV connector currently accepting `Licence` or `license`?
We probably want to make sure it matches the program behaviour (make sure the test cases are updated as well).
If it's hard to update the program behaviour, we probably can leave this one for now.

Reply:
Reverted this